programs/ssl/ssl_client2.c

```diff
@@ -984,6 +998,18 @@ int main( int argc, char *argv[] )
             if( opt.debug_level < 0 || opt.debug_level > 65535 )
                 goto usage;
         }
+#if defined(MBEDTLS_VERSION_C)
+        else if( strcmp( p, "build_version" ) == 0 )
+        {
+            if( strcmp( q, "1" ) == 0 )
+            {
+                mbedtls_printf( "build version: %s (build %u)\n",
+                                MBEDTLS_VERSION_STRING,
```

Comment:
How about mbedtls_version_get_string? and remove the

>  "#include "mbedtls/build_info.h""

Reply:
Agree, will modify and update a new patch soon.

Reply:
I have updated a new commit(076b2d06) to fix this. Please take a look.